### DISCUSSION TITLE
[Kavita] JPEG XL (.jxl) support for Reading

### DISCUSSION THREAD

**Discussion by fluff-li** [Kareadita/Kavita#3099] [Ideas]
### Idea Description

Support for scanning and displaying .jxl Images just like jpg/png and support .cb* archives with JPEG XL.

Why JPEG XL?
JPEG XL supports lossless conversion from old JPEG images keeping the same quality while saving space.

### Idea Category

API

### Duration of Using Kavita

On and Off for multiple Years  ~4 Years

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
Just to make my position clear on this request, until FF and Chrome add support, it is unlikely I will take on the overhead. I believe that end users should not need to know if an image format is supported by their browser or not when they use the software. Even if an admin sets it, it leaves a bad impression on users when things don't work perfectly. 

I also don't want to have to add transcoding of images on the fly to support JXL. I hope that with Safari adding support other browsers will follow. I know many people want this feature.

**Reply by therobbiedavis** [reply]
Keep an eye here: https://caniuse.com/jpegxl

**Reply by BPplays** [reply]
go to my comment in https://github.com/Kareadita/Kavita/discussions/2610

~@majora2007 would you accept a PR by someone that adds jxl support for just the reader and generating covers from (not into jxl from a jxl).
i think supporting reading jxl would be nice since mihon supports jxl, especially since i would guess most users are probably the only user in their instance so they could only mess it up for themselves~

**Reply by scare376** [reply]
This issue is closed because it is a duplicate. Please go to the open one.

**Comment by scare376** [comment]
This is also a duplicate of #2610
